generator/src/main/java/dev/cdevents/generator/CDEventsGenerator.java

```diff
@@ -92,38 +98,41 @@ private static void generateClassFileFromSchemaData(Mustache mustache, SchemaDat
 
     private static SchemaData buildCDEventDataFromJsonSchema(File file) {
         SchemaData schemaData = new SchemaData();
-
         log.info("Processing event JsonSchema file: {}", file.getAbsolutePath());
         try {
             JsonNode rootNode = objectMapper.readTree(file);
             JsonNode contextNode = rootNode.get("properties").get("context").get("properties");
             JsonNode subjectNode = rootNode.get("properties").get("subject").get("properties");
             String schemaURL = rootNode.get("$id").asText();
+            boolean isCustomEvent = schemaURL.endsWith("custom");
```

Comment:
This seems really fragile. So any URL that ends with `custom` is assumed to be `custom`? Do we define that anywhere? Basically if we have some other even name `"some/spec/that-has-custom"` would be skipped is it wasn't a custom event

While concerning, and I dont think it needs to be addressed in this PR, can you please add a comment or a TODO to make it less fragile? This may require work in the spec SIG to discuss how to easily identify custom events based on URL.

Reply:
No, not looking for custom event schema to check `isCustomEvent` here, generating a base `CustomTypeEvent.java` from an official [schema.json](https://github.com/cdevents/spec/blob/main/custom/schema.json), which has `"$id": "https://cdevents.dev/0.5.0-draft/schema/custom",`.
And this class file will be used to [create any custom events](https://github.com/Nordix/sdk-java/blob/custom_cdevents_support/docs/CUSTOM_CDEVENTS.md#create-a-custom-cdevent)

Reply:
We discussed this in the CDEvents WG and agreed that extending the match to `schema/custom` should be sufficient. WDYT?

Reply:
yeah sounds good to me.